README.md

```diff
@@ -1103,6 +1120,8 @@ git push heroku master
 Webpacker lazily compiles assets in test env so you can write your tests without any extra
 setup and everything will just work out of the box.
 
+Note, [React on Rails] users should set configuration value `compile` to false, as React on Rails
+handles compilation for test and production environments.
```

Comment:
I think all integration advice for React on Rails is best presented within the React on Rails documentation.